### DISCUSSION TITLE
CNCF TAG Runtime Discussion/Presentation

### DISCUSSION THREAD

**Discussion by rajaskakodkar** [opentofu/opentofu#739] [General]
Hello OpenTofu maintainers,

As the Tech Lead of [CNCF TAG Runtime](https://github.com/cncf/tag-runtime), I am reaching out to you to present/discuss OpenTofu at one of our meetings.

It would be great to learn about the open-source infrastructure as code tool and to know what lies on your roadmap. 

If you are interested, there is a slot available on 2nd Nov(just before [KubeCon + CloudNativeCon](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/)) and 16th Nov on our [agenda](https://docs.google.com/document/d/1k7VNetgbuDNyIs_87GLQRH2W5SLgjgOhB6pDyv89MYk/edit#heading=h.68c0ynema0xi). Please feel free to add it to the agenda or reach out to us in the [#tag-runtime](https://slack.cncf.io/) channel on the CNCF slack.

Thanks!

cc: @raravena80 @helayoty @nikhita

**Comment by cube2222** [comment]
Hey @rajaskakodkar!

Apologies for the late reply, this flew under our radar.

Would love to hop on the 16th November call and tell you a bit about OpenTofu.
